beard/clustering/blocking.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Beard.
+# Copyright (C) 2014 CERN.
+#
+# Beard is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Blocking for clustering estimators.
+
+.. codeauthor:: Gilles Louppe <g.louppe@cern.ch>
+
+"""
+import numpy as np
+
+from sklearn.base import BaseEstimator
+from sklearn.base import ClusterMixin
+from sklearn.base import clone
+from sklearn.utils import column_or_1d
+
+
+def _single(X):
+    return np.ones(len(X), dtype=np.int)
```

Comment:
If your intention was to reuse this function in future, shouldn't it be moved to a utils file?